source/_components/switch.raspihats.markdown

```diff
@@ -69,6 +69,11 @@ i2c_hats:
               required: false
               default: false
               type: boolean
+            initial_state:
+              description: "The initial state, can be either `true` or `false`. `none` means no state is forced on the corresponding digital output when this switch is instantiated."
+              required:
```

Comment:
`required: false`

Offtopic: anyone got those ['Suggested Changes'](https://blog.github.com/2018-10-16-future-of-software/?utm_source=github&utm_medium=banner&utm_campaign=ww-dotcom-universe-20180919&utm_content=dash-blog#suggested-changes-public-beta) working?